src/components/Calendar/PCalendar.vue

```diff
@@ -68,6 +68,7 @@
 .p-calendar__dates { @apply
   grid
   grid-cols-7
+  grid-rows-6
```

Comment:
so this tailwind class makes the css `grid-template-rows: repeat(6, minmax(0, 1fr))`. I believe the `minmax` is the important part and not the 6. I think you can use `grid-auto-rows: minmax(0, 1fr)`

Reply:
@pleek91 that is true, that means rows fill the space available, assuming we prefer that over set height regardless of viewing month?

Reply:
I think so? That leaves the existing layout but fixes it for safari right? Again, I'm unable to reproduce.

Reply:
@stackoverfloweth

Reply:
applied suggestion, thanks